src/components/LoginPasswordForm.vue

```diff
@@ -142,6 +142,19 @@ export default defineComponent({
       submit,
       removePassword
     }
+  },
+  watch: {
+    showPassword() {
+      const inputElement = this.passwordField.$el.querySelector('input');
+      if (inputElement) {
+        const cursorPosition = inputElement.selectionStart || 0;
+
+        requestAnimationFrame(() => {
+          inputElement.setSelectionRange(cursorPosition, cursorPosition);
+          inputElement.focus();
+        });
+      }
+    }
```

Comment:
Don't use `watch` property, it's a Options API. Use hook https://github.com/Adamant-im/adamant-im/pull/747#discussion_r2010863256